build.sh

```diff
@@ -1,10 +1,10 @@
 docker build -t milideluxe/s3-imaginary -f prod.Dockerfile .
```

Comment:
The tag doesn't have to be in in 'name:tag' format for 'milideluxe/s3-imaginary'?
https://docs.docker.com/v17.12/engine/reference/commandline/build/#options